packages/@okta/i18n/src/properties/login.properties

```diff
@@ -832,6 +833,14 @@ oie.numberchallenge.warning = Haven't received a push notification yet? Try open
 ## {0} refers to a number which is the correct answer for the number challenge
 oie.numberchallenge.instruction = On your mobile device, open the Okta Verify prompt, then tap <$1>{0}</$1> in Okta Verify to continue.
 
+## IdP Authenticator
+# {0} refers to the display name of the authenticator for all IdP Authenticator values
+oie.customFactor.authenticator.description=Redirect to verify with {0}
```

Comment:
should we change also the keys to `oie.idpAuthenticator.*`? Just to keep the same names everywhere and not confuse any future dev

Reply:
Updated to `oie.idp` to be consistent with other factors!